cmd/cmd_utils.go

```diff
@@ -704,12 +704,36 @@ func showUsageExample(cmd *cobra.Command, details string) {
 	u.PrintInvalidUsageErrorAndExit(errors.New(details), suggestion)
 }
 
+// filterStackNames filters out file paths and returns valid stack names
+func filterStackNames(stacksMap map[string]any, basePath string) []string {
```

Comment:
Let’s revise this function name. We want to add proper filtering later (not this PR), but this is filtering on paths. Also, it’s not clear to me why we need this. Can you add a comment explaining why we need this?